scenes/controller/unlock_controller/unlock_controller.gd

```diff
@@ -1,5 +1,8 @@
 extends Node
 
+@export var sfx_unlock: AudioStream
+
+@onready var firepit_timer: Timer = $FirepitTimer
```

Comment:
`firepit_timer` not used

Reply:
This was from merging in master, are you saying I should remove it?

Reply:
Yes, looks like it is not used anymore _(I refactored this firepit mechanic to be consistent with house mechanic in worker controller)._